.gitignore

```diff
@@ -49,6 +49,7 @@ test-driver
 erl_crash.dump
 project.lock.json
 
+.Dockerfile.sha512
```

Comment:
wondering what will generate this?

Reply:
The `build/docker/refresh.sh` command from here https://github.com/apache/thrift/tree/master/build/docker Although it seems like we're moving away from Travis CI / Docker anyways so doesn't matter too much